CONTRIBUTING.md

```diff
@@ -8,10 +8,8 @@ Pull requests will be accepted based on the following:
 1. If the company is at least partially remote
 1. If you add the company in alphabetical order in the list
 1. If you submit the company with a website
-1. If you submit the company with the regions that are accepted for remote positions (US only? Worldwide? Specific timezones?) \*
-1. If you submit a company profile page and link the company name to it (see example [here](/company-profiles/example.md))
-
-\* Optional, but highly recommended
+1. If you submit the company with the regions that are accepted for remote positions (for more details see the instructions in the [example company profile](/company-profiles/example.md))
+1. If you submit a company profile page and link the company name to it (see example [here](/company-profiles/example.md#region))
```

Comment:
I wonder if this needs reworded. Reading it over, it is confusing. 

What this line is saying is this - you create a company profile file and you also link it from the main readme.md file 

That example shouldn't be linked to a section of the example.md, but also example.md doesn't help illustrate the point. 
Wondering if a screenshot may help.

Reply:
Yep, agreed.  Right now in this PR, the "If you submit a company profile page and link the company name to it" link has the `#region` anchor, but the more specific "region" link just links to the whole example profile.

To keep this PR focused, though, I think we should save any changes to the "submit a company profile page and link the company name to it" step for a later PR.

Reply:
Sorry guys, this is my fault. @nylen told me to use the `#region` anchor for bullet point no.5 in the list, which is:  

*If you submit the company with the regions that are accepted for remote positions (for more details see the instructions in the [example company profile](/company-profiles/example.md))* 

That totally makes sense as somewhere to put the `#region` link, but I incorrectly added it to no.6 about the company name on the readme.md. Sorry!!!

@dougaitken, yes a screenshot sounds like a good idea. I've also made this mistake when changing a company profile in the past so I think it would really help. What do you think should be on the screenshot? The readme.md file with an arrow pointing to where you should change the company name? 😄 

@nylen agreed this should be for a later PR.